src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -157,6 +159,16 @@ public async Task RunAsync()
             }
         }
 
+        public (string, bool, bool) GetNodeFolder(IExecutionContext executionContext)
```

Comment:
public string GetNodeLocation()
{
bool useNode10 = ExecutionContext.Variables.GetBoolean("AGENT_USE_NODE10") ?? StringUtil.ConvertToBoolean(System.Environment.GetEnvironmentVariable("AGENT_USE_NODE10"), false);

bool taskHasNode10Data = Data is Node10HandlerData;
 string nodeFolder = (taskHasNode10Data || useNode10) ? "node10" : "node";
Trace.Info($"Task.json has node10 handler data: {taskHasNode10Data}, use node10 for node tasks: {useNode10}");	

return file = Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Externals), nodeFolder, "bin", $"node{IOUtil.ExeExtension}");
}